packages/mobile/src/components/Chat/Chat.stories.tsx

```diff
@@ -221,20 +221,6 @@ storiesOf('Chat', module)
                 pubKey: 'test',
               },
             ],
-            [
```

Comment:
Why did you remove this long message from stories?

Reply:
Because it brings no value and makes the story messy

Reply:
could you add a comment explaining this? It's indeed misleading and tempting to remove unless you know the whole story behind it - it's testing proper line breaking